docs/man_pages/project/creation/sample-native.md

```diff
@@ -0,0 +1,31 @@
+sample native
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder sample native`    
+
+Lists all available NativeScript sample apps with name, description, GitHub repository, and clone command. To clone a selected sample app,
```

Comment:
Lists all available NativeScript sample apps with name, description, GitHub repository and clone command. To clone a selected sample app, run its clone command.